src/store/MultiStore/MultiStore.tsx

```diff
@@ -0,0 +1,39 @@
+import { makeObservable, observable, action } from 'mobx';
+import { Option } from '../../components/MultiDropdown/MultiDropdown';
+
+class MultiStore {
+
+    tags: Option[] = []
+    selectedTags: Option[] = []
+    constructor() {
+        makeObservable(this, {
+            tags: observable,
```

Comment:
```suggestion
            tags: observable.ref,
```

тк ты не изменяешь элементы массива, а просто меняешь ссылку на массив